parser.go

```diff
@@ -1489,6 +1357,95 @@ func (p *parser) getTypeAsString(fieldType interface{}) string {
 	return fmt.Sprint(fieldType)
 }
 
+func parseStructTags(astField *ast.Field, structSchema *SchemaObject, fieldSchema *SchemaObject, name string) (newName string, skip bool) {
```

Comment:
did some refactoring, i didn't write this. just extracted that super long method into a slightly shorter but still long function.